translator/source_code/cotrip_translator.py

```diff
@@ -0,0 +1,294 @@
+import json
+from datetime import datetime, timezone, timedelta
+import uuid
+import random
+import string
+import sys, getopt
```

Comment:
Several of these imports are unused, can we clean up?

Reply:
i removed the unused imports and that extension worked very well.

Reply:
Still quite a few unused imports.

Reply:
deleted those imports

Reply:
`cotrip_translator.py` still has a couple unused imports:
![image](https://user-images.githubusercontent.com/2932159/118304491-f5b6d100-b4a3-11eb-940c-9eeb437b7e30.png)

Here it appears that "ge" and "validate" are not used.

Reply:
deleted those.should be fixed now